core/src/main/java/io/aiven/klaw/controller/AclController.java

```diff
@@ -232,4 +233,17 @@ public ResponseEntity<ServiceAccountDetails> getAivenServiceAccountDetails(
   public ResponseEntity<Set<String>> getAivenServiceAccounts(@RequestParam("env") String envId) {
     return new ResponseEntity<>(aclControllerService.getAivenServiceAccounts(envId), HttpStatus.OK);
   }
+
+  /**
+   * @param aclRequestId requestId of Acl
```

Comment:
I think you need to add the openapi.yaml and add the entry into the api.ts file as well in coral to include this.